src/components/WorkersTable.vue

```diff
@@ -11,8 +11,12 @@
         <span>{{ row.name }}</span>
       </template>
 
-      <template #last-polled="{ value }">
-        <span>{{ formatDateTimeNumeric(value) }}</span>
+      <template #last-seen="{ value }">
+        <span>{{ formatDateTimeRelative(value) }}</span>
```

Comment:
We do have a composition called `useNow` which you can use with `formateDateTimeRelative` to get a dynamically updating time. We used that for work pools [here](https://github.com/PrefectHQ/prefect-ui-library/blob/main/src/compositions/useWorkPoolLastPolled.ts#L53)

Might be cool to do the same here.